4-projects/modules/standard_projects/data.tf

```diff
@@ -31,3 +31,12 @@ data "google_compute_network" "nonprod_shared_vpc" {
   name    = "shared-vpc-nonprod"
   project = local.nonprod_project_id
 }
+
+data "google_projects" "dev_project" {
+  filter = "labels.application_name:org-shared-vpc-dev"
```

Comment:
The values for `labels.application_name` and `google_compute_network.name` used in this file are no longer valid.

Current values can be found in the files below:

-  https://github.com/terraform-google-modules/terraform-example-foundation/blob/develop/2-environments/modules/env_baseline/networking.tf
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/develop/3-networks/modules/restricted_shared_vpc/main.tf
- https://github.com/terraform-google-modules/terraform-example-foundation/blob/develop/3-networks/modules/standard_shared_vpc/main.tf

Reply:
This feedback is still valid for the `single_project` module